indsl/signals/user_specified_timeseries.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Cognite AS
+from typing import List
+
+import pandas as pd
+
+from indsl.exceptions import UserValueError
+
+
+# from indsl.validations import validate_series_is_not_empty
+
+
+# @check_types
+def user_specified_timeseries(
+    input_time: List[float] = [0.0, 1.0], input_values: List[float] = [0.0, 1.0]
+) -> pd.Series:
+    """Convert input to a timeseries.
```

Comment:
How would this work in the UI? user manually copy pastes all the values into the parameters of the nodes?